bpython/curtsiesfrontend/repl.py

```diff
@@ -1060,10 +1151,10 @@ def toggle_file_watch(self):
             )
 
     # Handler Helpers
-    def add_normal_character(self, char):
+    def add_normal_character(self, char, add_to_search=True):
```

Comment:
Could we call these "use_to_narrow_search" or "narrow_search" instead? `add_to_search` sounds to me like we're adding it to history.

Reply:
okay i will change that right away. np with that